common/src/main/scala/com/gu/recipeasy/db/actions.scala

```diff
@@ -0,0 +1,47 @@
+package com.gu.recipeasy.db
+
+import java.sql.Types
+import java.time.OffsetDateTime
+import java.util.Date
+
+import com.gu.recipeasy.models._
+import io.circe.generic.auto._
+import io.circe.syntax._
+import io.getquill._
+import org.postgresql.util.PGobject
+
+import scala.reflect.ClassTag
+
+object db {
```

Comment:
I would call this `object DB` (in uppercase) and rename the file to `DB.scala`.

Once we have more DB tables, we'll probably want to have separate DAOs per table (`Recipes.scala`, `Users.scala`, etc.) but we'll deal with that later.